types/tampermonkey/tslint.json

```diff
@@ -0,0 +1,6 @@
+{ 
+  "extends": "dtslint/dt.json",
+  "rules": {
+    "no-unnecessary-generics": false
```

Comment:
I don't see anything that looks like an unneeded generic - why's this lint rule need to be ignored?

Reply:
It's here
```
declare function GM_xmlhttpRequest<TContext = any>(
    details: Tampermonkey.Request<TContext>
): Tampermonkey.AbortHandle<void>;
```

Use case
```
interface RequestContext {
    form: {
        name: string
    };
}

GM_xmlhttpRequest<RequestContext>({
    url: 'http://example.com/',
    onloadstart(response) {
        GM_log(response.context.form.name);
    },
});
```

Reply:
I'd prefer to see an inline suppression on that instance than a blanket suppression in the `.json` file, if you could, then - that way future editors don't accidentally add more they don't need to.

Reply:
Sure, I'll add an inline suppression